docs/src/cli/index.md

```diff
@@ -0,0 +1,103 @@
+# WEBKNOSSOS CLI
+
+Python CLI for creating and working with [WEBKNOSSOS](https://webknossos.org/) [WKW](https://github.com/scalableminds/webknossos-wrap) datasets. WKW is a container format for efficiently storing large, scale 3D image data as found in (electron) microscopy.
+
+## Features
+
+The WEBKNOSSOS CLI offers many useful commands to work with WEBKNOSSOS datasets:
+
+- `webknossos compress`: Compress a WEBKNOSSOS dataset
+- `webknossos convert`: Convert an image stack (e.g., `tiff`, `jpg`, `png`, `bmp`, `dm3`, `dm4`) to a WEBKNOSSOS dataset
+- `webknossos convert-knossos`: Converts a KNOSSOS dataset to a WEBKNOSSOS dataset
+- `webknossos convert-nifti`: Converts a NIFTI image or a directory with NIFTI files into a WEBKNOSSOS dataset
+- `webknossos convert-raw`: Converts a RAW image file to a WEBKNOSSOS dataset
+- `webknossos convert-zarr`: Converts a Zarr dataset to a WEBKNOSSOS dataset 
+- `webknossos download`: Download a dataset from a WEBKNOSSOS server as WKW format
+- `webknossos downsample`: Downsample a WEBKNOSSOS dataset
+- `webknossos upload`: Upload a local WEBKNOSSOS dataset to a remote location
+- `webknossos upsample`: Upsample a WEBKNOSSOS dataset
+
+## Supported input formats
+
+- Standard image formats, e.g. `tiff`, `jpg`, `png`, `bmp`
+- Proprietary image formats, e.g. `dm3`
+- Raw binary files
+
+## Installation
+
+### Python 3 with pip from PyPi
+
+- `webknossos` requires at least Python 3.8
+
+```bash
+pip install webknossos
```

Comment:
With extras, like this?
```suggestion
pip install webknossos[all]
```

Reply:
I have changed it to `pip install "webknossos[all]".

Reply:
@markbader Do you have access to a Windows machine by any chance to test this? The `[all]` parameter will install the BioFormats package, correct? Is that Windows compatible?